### PR TITLE
docs: update input/hidden

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Update input/hidden content with newest EN document

### CODE REVIEW

File: files/zh-cn/web/html/element/input/hidden/index.md [hunk 1]
```diff
@@ -5,112 +5,103 @@ slug: Web/HTML/Element/Input/hidden
...
 
 在这里的步骤 2 思路中，要被更新的 ID 记录保留在隐藏的输入中。在步骤 3 中提交表单后，该 ID 随记录内容自动发送回服务器。该 ID 使站点的服务器端组件确切知道哪些记录需要被提交的数据更新的。
 
-您可以在下面的 [例子](/zh-CN/docs/Web/HTML/Element/input/hidden#例子) 部分中看到完整的[示例](/zh-CN/docs/Web/HTML/Element/input/hidden#例子)。
+你可以在下面的[示例](#示例)部分中看到完整的示例。
 
-### 改善网站安全性
+### 提升网站安全性
 
 隐藏输入表单还用于存储和提交安全令牌或机密信息，以提高网站的安全性。基本思路是，如果用户填写敏感表格，例如在其银行网站上将某笔款项转入另一个帐户的表格，他们将被提供的密钥和证明他们就是他们所说的真实身份，并且他们使用正确的表单来提交转移请求。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/input/hidden/index.md:73]:
```suggestion
### 提升网站安全性

隐藏输入表单还用于存储和提交安全令牌或机密信息，以提高网站的安全性。基本思路是，如果用户正在填写敏感的表单，例如在银行网站上将某笔款项转入另一个账户的表格，所提供的机密信息将证明他们的身份是真实的，并且使用了正确的表单来提交转账请求。
```

File: files/zh-cn/web/html/element/input/hidden/index.md [hunk 2]
```diff
@@ -5,112 +5,103 @@ slug: Web/HTML/Element/Input/hidden
...
   </div>
   <div>
-    <label for="content">Post content:</label>
+    <label for="content">文章内容：</label>
     <textarea id="content" name="content" cols="60" rows="5">
 This is the content of my excellent blog post. I hope you enjoy it!
     </textarea>
   </div>
   <div>
     <button type="submit">Update post</button>
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/input/hidden/index.md:102]:
```suggestion
    <button type="submit">更新文章</button>
```

File: files/zh-cn/web/html/element/input/hidden/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/html/element/input/hidden/index.md:73]: 👍

Comment (yin1999) [reply files/zh-cn/web/html/element/input/hidden/index.md:73]: 请更新

**Review by yin1999 — APPROVED** [review]

**Pull request merged**